=== RUN   TestMixed2
--- FAIL: TestMixed2 (0.00s)
panic: sync: negative WaitGroup counter [recovered]
	panic: sync: negative WaitGroup counter

goroutine 6 [running]:
testing.tRunner.func1.2({0x5488c0, 0x5b16c0})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1734 +0x21c
testing.tRunner.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1737 +0x35e
panic({0x5488c0?, 0x5b16c0?})
	/home/ilian/Tools/go-runtime/src/runtime/panic.go:792 +0x132
sync.(*WaitGroup).Add(0xc00007e760?, 0x4daa93?)
	/home/ilian/Tools/go-runtime/src/sync/waitgroup.go:64 +0xcd
sync.(*WaitGroup).Done(...)
	/home/ilian/Tools/go-runtime/src/sync/waitgroup.go:89
scenarios.TestMixed2(0xc000003880?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_test.go:30 +0x2c
testing.tRunner(0xc000003880, 0x585d88)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	0.006s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_test.go
TestName:  TestMixed2
Import added at line: 3
Header added at line: 27
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_mixed_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestMixed2

Exit Replay with code  34 Negative WaitGroup counter
Exit Message: panic: sync: negative WaitGroup counter

FAIL	scenarios	0.106s
FAIL
Bugreport info: 0_1,fail
Bugreport info: 0_2,fail
